renovate.json

```diff
@@ -34,6 +33,14 @@
       "matchDepTypes": ["devDependencies"],
       "rangeStrategy": "bump"
     },
+    {
+      "matchDepTypes": ["devDependencies"],
+      "groupName": "all major dev dependencies",
+      "groupSlug": "all-major-dev",
+      "matchUpdateTypes": [
+        "major"
+      ]
+    },
```

Comment:
I prefer to not group major dev dependencies as the major bump indicate that they are breaking in some way and bundling many such together can make it harder to merge as a blocking breaking change from any one dependency will block the merging of them all